packages/wonder-blocks-accordion/src/components/accordion-section.tsx

```diff
@@ -227,32 +251,51 @@ const AccordionSection = React.forwardRef(function AccordionSection(
                 isFirstSection={isFirstSection}
                 isLastSection={isLastSection}
             />
-            {/* The content is the section that expands and closes. */}
-            {expandedState ? (
-                <View
-                    id={sectionContentUniqueId}
-                    style={[
-                        styles.contentWrapper,
-                        sectionStyles.contentWrapper,
-                    ]}
-                >
-                    {typeof children === "string" ? (
-                        <Body style={styles.stringContent}>{children}</Body>
-                    ) : (
-                        children
-                    )}
-                </View>
-            ) : null}
+            <View
+                id={sectionContentUniqueId}
+                style={[
+                    styles.contentWrapper,
+                    expandedState
+                        ? styles.contentWrapperExpanded
+                        : styles.conentWrapperCollapsed,
+                    sectionStyles.contentWrapper,
+                ]}
+            >
+                {typeof children === "string" ? (
+                    <Body style={styles.stringContent}>{children}</Body>
+                ) : (
+                    children
+                )}
+            </View>
         </View>
     );
 });
 
 const styles = StyleSheet.create({
     wrapper: {
-        flexDirection: "column",
+        // Use grid layout for more performant animation
+        display: "grid",
         boxSizing: "border-box",
     },
+    wrapperWithAnimation: {
+        transition: "grid-template-rows 300ms",
+    },
+    wrapperCollapsed: {
+        gridTemplateRows: "min-content 0fr",
+    },
+    wrapperExpanded: {
+        gridTemplateRows: "min-content 1fr",
+    },
     contentWrapper: {
+        overflow: "hidden",
+    },
+    conentWrapperCollapsed: {
+        // Make sure screen readers don't read the content when it's
+        // collapsed.
+        visibility: "hidden",
+    },
```

Comment:
**suggestion:** this style should be applied after the animation completes.  Currently, it gets applied at the start which causes the contents to pop out of existence.

Reply:
Oh sorry! This isn't the correct approach. Let me push that really quickly.